clients/egui/src/account/mod.rs

```diff
@@ -418,6 +437,31 @@ impl From<SyncUpdate> for AccountUpdate {
     }
 }
 
+fn start_auto_syncing(
+    settings: &Arc<RwLock<Settings>>, core: &Arc<lb::Core>,
+    update_tx: &mpsc::Sender<AccountUpdate>, ctx: &egui::Context,
+) {
+    let settings = settings.clone();
+    let core = core.clone();
+    let update_tx = update_tx.clone();
+    let ctx = ctx.clone();
+
+    thread::spawn(move || loop {
+        if settings.read().unwrap().auto_sync {
+            match core.calculate_work() {
```

Comment:
I think calling calculate work every instant may not be good since it does a network call each time. I'm not too sure what's the best duration between calls, I know on android its 30 seconds, although it should maybe be something like 1 or 2 minutes.

Reply:
True, I'll rfc this